docs/topics/i18n/translation.txt

```diff
@@ -70,6 +70,16 @@ as a shorter alias, ``_``, to save typing.
        global ``_()`` function causes interference. Explicitly importing
        ``ugettext()`` as ``_()`` avoids this problem.
 
+    .. warning::
```

Comment:
Here's my suggested wording:
```
.. admonition:: What functions may be aliased as ``_``?

    Only :func:`~django.utils.translation.gettext`,
    :func:`~django.utils.translation.gettext_lazy`,
    :func:`~django.utils.translation.ugettext`, and
    :func:`~django.utils.translation.ugettext_lazy` can be imported as
    ``_``. :djadmin:`makemessages` ignores strings wrapped with any other
    function such as :func:`~django.utils.translation.pgettext`, even if it's
    aliased as ``_``.
```
Would like @claudep to confirm it's correct though.